salt/init.sls

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+include:
+  - salt.pkgrepo
+      {%- if salt.config.get('salt_formulas:list') %}
+  - salt.formulas
+      {%- endif %}
+      {%- if salt.config.get('salt:master')|length > 1 %}
+  - salt.master
+      {%- endif %}
+      {%- if salt.config.get('salt:cloud')|length > 1 %}
+  - salt.cloud
+      {%- endif %}
+      {%- if salt.config.get('salt:ssh_roster') %}
+  - salt.ssh
+      {%- endif %}
+  - salt.standalone
+  - salt.minion
```

Comment:
There are situations where you wouldn't want `standalone` or even `minion` to be run, such as only wanting `ssh` or a `master`.  So these will need `if` blocks as well.

Reply:
done.